ln-simln-jamming/src/analysis.rs

```diff
@@ -57,17 +57,32 @@ impl Serialize for Record {
                 .forwarding_outcome(self.forward.amount_in_msat, self.forward.incoming_endorsed),
         )?;
         state.serialize_field(
-            "revenue_threshold",
-            &self.decision.reputation_check.revenue_threshold,
+            "incoming_revenue",
```

Comment:
incoming_revenue/incoming_revenue_threshold + make `in_flight_risk` directional and write for both

I think we can also go ahead and add `outgoing_revenue_threshold` and `incoming_reputation`, all the values are useful here 👍